src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
+        <fullName>Recurring_Donation_Batch_Size__c</fullName>
+        <defaultValue>50</defaultValue>
+        <description>The number of records processed at a time when running the Recurring Donations job. The default size is 50. Reduce to a smaller number if the Recurring Donation job is failing due to system limits.</description>
```

Comment:
@smarovich Would you mind taking a look at the description/inline help text for this new custom settings field? It's not visible in the NPSP Setting UI yet, but it will be soon.

Reply:
@force2b  @jesslopez-sf   Looks great. I'd make some very minor changes:  The number of records to process at a time when running the Recurring Donations batch job. The default size is 50. Reduce to a smaller number if the Recurring Donations batch job is failing due to system limits.

Reply:
Thanks, @smarovich ! I updated the label with your suggestions.